formats/AtomFormat.php

```diff
@@ -164,7 +161,10 @@ public function display(){
 		return parent::display();
 	}
 
+	/**
+	 * @deprecated Use e() instead
+	 */
 	private function xml_encode($text){
```

Comment:
If we wanted to switch to another function, we can just drop this since it is private.